Source/Documentation/Manual/physics.rst

```diff
@@ -3166,6 +3168,28 @@ To control the application and release rates on the brake use the ``EngineBrakes
 The ``SteamBrakeFX`` special effect, if added to the wagon, will turn on and off with the brake operation 
 and can be used to model steam leakage of the steam brake cylinder, etc.
 
+Wheel Slide Protection
+----------------------
+
+Open Rails supports the use of Wheel Slide Protection (WSP) on trains with air brakes. WSP operates as described below.
+
+During braking wheelslide control is effected throughout the train by additional equipment on each vehicle. In the piping to each 
+pair of brake cylinders are fitted electrically operated dump valves. When axle rotations which are sensed electrically, differ 
+by a predetermined speed the dump valves are operated releasing brake cylinder pressure to both axles of the affected bogie.
```

Comment:
How does the player know that the valve has operated?
Would it be appropriate to add a "confirmer" announcement here?

Reply:
Thanks for the comment.

We will see what comes out of the forum discussion.